content/compose/install/standalone.md

```diff
@@ -50,12 +50,11 @@ on Microsoft Windows Server and want to install Docker Compose.
 3. Run the following command to download the latest release of Compose ({{% param "compose_version" %}}):
 
     ```powershell
-     Start-BitsTransfer -Source "https://github.com/docker/compose/releases/download/{{% param "compose_version" %}}/docker-compose-windows-x86_64.exe" -Destination $Env:ProgramFiles\Docker\docker-compose.exe
+     Start-BitsTransfer -Source "https://github.com/docker/compose/releases/download/{{% param "compose_version" %}}/docker-compose-windows-x86_64.exe" -Destination $env:windir\System32\docker-compose.exe
```

Comment:
Not sure if this is correct; this would change the compose CLI to be installed in the System32 directory (e.g. `C:\Windows|System32`), which doesn't look like the correct location for this.